### DISCUSSION TITLE
Does GitHub Actions support Discussions already?

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#3117] [Discussions] [answered]
I was wondering, if GitHub Actions do support Discussions, as it could be quite useful for things like serving canned responses to commonly asked questions.

For example, if user A askes a question that is answered in the FAQ and if you have a action that listens for message creation/sending, could the Action react to something like `@github-actions pls answer faq 1` and such a response would be posted:
```
Hello @user,

Your question is already being answered in out faq, so you may want to take a look at it: https://example.com/faq
More specifically can the answer to your question be found here: https://example.com/faq#1
```

**Comment by evi-liu** [comment, marked as answer]
This is now available in private beta! For details and how to test it out, see https://github.com/github/feedback/discussions/4173
